extensions/civetweb/processors/ListenHTTP.h

```diff
@@ -205,13 +193,17 @@ class ListenHTTP : public core::Processor {
   void notifyStop() override;
 
  private:
-  // Logger
-  std::shared_ptr<logging::Logger> logger_;
+  static const std::size_t DEFAULT_BUFFER_SIZE;
 
+  void processIncomingFlowFile(core::ProcessSession *session);
+  void processRequestBuffer(core::ProcessSession *session);
+
+  std::shared_ptr<logging::Logger> logger_;
   CivetCallbacks callbacks_;
   std::unique_ptr<CivetServer> server_;
   std::unique_ptr<Handler> handler_;
   std::string listeningPort;
+  std::size_t batch_size_;
```

Comment:
I know it will be set in `onSchedule()`, but I think it would be better to initialize `batch_size_`, either here or in the constructor.

Reply:
Fixed in [7813ee3](https://github.com/apache/nifi-minifi-cpp/pull/921/commits/7813ee3aa9fc16db0b132d49f27d88bf4671df3f)